content/hardware/05.pro-solutions/solutions-and-kits/edge-control/tutorials/smart-irrigation-system/content.md

```diff
@@ -0,0 +1,533 @@
+  water_volume = PI * (radius) * (radius) * ((int)water_level/100) * 1000;
+
+  if (waterCtrl == 0) {
+    currentWater = water_volume;
+    Serial.println("Initial Water Volumen: ");
+    Serial.println(currentWater);
+    water_usage = 0;
+    waterCtrl = 1;
+  }
+
+  if (water_volume < currentWater) {
+    water_usage += currentWater - water_volume;
+    currentWater = water_volume;
+  } else if (water_volume > currentWater) {
+    currentWater = water_volume;
+  }
+```
+
+### The Arduino IoT Cloud Dashboard
+
+Taking advantage of the Arduino IoT Cloud, its possible to seamlessly integrate a simple but powerful dashboard to monitor and visualize the status of the system in real-time, resulting in a professional Human-Computer Interaction (HCI) as can be seen below:
+
+![Arduino Cloud project dashboard ](assets/dashboard_16-9.png)
+
+Within the Arduino IoT Cloud's dashboard, the system variables can be monitored and controlled. There are temperature, humidity and rain probability gauges showing the current weather status and forecast, toggle switches to control each motorized valve accompanied by a scheduler widget to set automatic irrigation routines, also time series graphs showing the daily activated time of each valve. Finally, there are water supply monitoring widgets, one that shows the current water level in percentage from 0-100%, a water volume widget that shows the remaining liters of water in the tank, and a water usage widget that shows the liters used daily.
+
+![Arduino Cloud project dashboard on a smartphone](assets/dashboard-cel.png)
+
+The dashboard can be easily accessible from a PC, mobile phone or tablet from anywhere, receiving an instantaneous update wherever we are.
+
+## Full Smart Irrigation System Example
+
+All the necessary files to replicate this application note can be found below:
+
+- The complete code can be downloaded [here](assets/Edge-Control_MKR_Codes.zip).
+
+### The Irrigation System Working
+
+Below you can find some additional images and animations showing how the system works:
+
+![Zone one manual irrigation](assets/manual-irrigation.png)
+![Automatic irrigation timer countdown in zone two (1.4 minutes left)](assets/timer-LCD.png)
+![Zone two after the motorized valve is opened](assets/zone2.png)
+![Irrigation in all zones](assets/4zones.gif)
+
+## Conclusion
+
+In this application note, we have learned how to build a smart irrigation system to water our crops automatically, manually and remotely. Thanks to real-time weather analysis, we can avoid irrigation when it's raining saving water and avoiding over-irrigation or flooding problems. 
+
+Arduino Edge Control allows us to easily implement this kind of agriculture systems ready for field deployment. It is known that normally in remote areas Wi-Fi® access is limited by its use is increasing in the creation of new smart farms and the connectivity can be extended using alternatives boards like the Arduino MKR Lora board.  
```

Comment:
```suggestion
Arduino Edge Control allows you to easily implement this kind of agriculture systems ready for field deployment. However, if Wi-Fi® access is limited in your area, the connectivity can be extended using alternative boards, like the [Arduino MKR WAN 1310](https://docs.arduino.cc/hardware/mkr-wan-1310).  
```

Reply:
thanks for the link to tthe right product :)

reminder for myself: about however expressions